tests/suites/test_suite_ssl.function

```diff
@@ -224,34 +224,36 @@ int mbedtls_mock_socket_connect( mbedtls_mock_socket* peer1,
 {
     int ret = -1;
```

Comment:
Should we set `peer1->input` and `peer2->input` to `NULL` before beginning memory allocation? The risk is if we can't guarantee these would be `NULL` at this point, we may call `mbedtls_mock_socket_close()` with `peer2->input` being an uninitialized garbage pointer (that shouldn't be used with `free()`).

Reply:
We do not have to set those inputs as NULL here because they should be set before connection, inside mbedtls_mock_socket_init.